templates/tpos/tpos.html

```diff
@@ -864,6 +866,9 @@ <h5 class="q-mt-none">
       atm() {
         if (this.withdrawamtposs > 0) {
           this.atmBox = true
+          if (this.atmPremium > 0) {
+            this.exchangeRate = this.exchangeRate * (1 - this.atmPremium)
```

Comment:
I think the correct formula is:

```suggestion
            this.exchangeRate = this.exchangeRate / (1 + this.atmPremium)
```